### PR TITLE
Story/003 list courses

### PR DESCRIPTION
**Pull request by e-mccoy**

### CODE REVIEW

**Review by ChrisThompsonK — APPROVED** [review]
Looks good to me

File: ExpressAPI/db.js [hunk 1]
```diff
@@ -10,4 +10,16 @@ const db = mysql.createConnection({
 db.connect(function(err){
     if(err) throw err;
     console.log("Connected to MySql");
-})
\ No newline at end of file
+})
+
+exports.getCourses = function(callback) {
```
Comment (JamesMatchett) [inline ExpressAPI/db.js:15]: yeah aware of this double effort between Eoin and myself so will just need to remember that when merging the later API stuff

File: training-hub/src/app/app.module.ts [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
   declarations: [
-    AppComponent
+    AppComponent,
+    CourseListComponent
   ],
   imports: [
-    BrowserModule
+    NgbModule,
+    BrowserModule,
+    HttpClientModule,
```
Comment (JamesMatchett) [inline training-hub/src/app/app.module.ts]: remove comma from end of this line

**Review by JamesMatchett — COMMENTED** [review]
fix the one comma issue

**Pull request closed without merging**